pkgs/tools/networking/ndjbdns/default.nix

```diff
@@ -1,25 +1,30 @@
-{ stdenv, fetchurl, systemd, pkgconfig }:
+{ stdenv, fetchFromGitHub, autoreconfHook, systemd, pkgconfig }:
+
+with stdenv.lib;
 
 stdenv.mkDerivation rec {
   version = "1.06";
```

Comment:
Looks to me like the delta contains only minor changes, primarily related to the build process or compat.